ecosystem/sep-0006.md

```diff
@@ -415,7 +435,8 @@ The response should be a JSON object like:
     "enabled": false
   },
   "transactions": {
-    "enabled": true
+    "enabled": true, 
```

Comment:
Should we add `"enabled": true` to the `"transaction"` info as well? Is there any case where `/transactions` requires authentication, but `/transaction` doesn't?

Reply:
I don't think there's a way to populate the `/transaction` request without a prior call to `/transactions`, because the first request requires at least one of a specific transaction's three identifiers. I'm not sure what this implies for authentication - but it's extremely unlikely, absent a very large DoS, that requesting `/transaction` without the prior authenticated call to `/transactions` would succeed.

Reply:
Yeah, I agree that it's unlikely.

In any case, I think it's still worth it to make the `/transaction` and `/transactions` calls auth criteria consistent, since if an anchor doesn't want to disclose information to non-authenticated sources about the whole transaction list, it probably wouldn't want do it for a specific transaction either (regardless of how hard it is to guess their identifier). Does that make sense?

Reply:
Yup, it does make sense! Agree that it's worth adding.